packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/index.tsx

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import AppLayout from '../../../components/AppLayout';
+import { HeadingPrimary } from '../../../components/HeadingPrimary';
+import FormChoice, {
+  CHOICES,
+  FormChoiceData,
+  FormChoiceOption,
+} from '../../../components/FormChoice';
+import { useFtlMsgResolver } from '../../../models';
+import {
+  BackupAuthenticationCodesImage,
+  BackupRecoveryPhoneSmsImage,
+} from '../../../components/images';
+import ButtonBack from '../../../components/ButtonBack';
+
+const SigninRecoveryMethod = ({}) => {
+  const onSubmit = async ({ choice }: FormChoiceData) => {
+    // TODO: actually do something with this, maybe pull into container
+    console.log('Submitted with choice:', choice);
+  };
+
+  const ftlMsgResolver = useFtlMsgResolver();
+  // TODO, actually pull these values
+  const numberOfCodes = 4;
+  const lastFourPhoneNumber = 3019;
+
+  const formChoices: FormChoiceOption[] = [
+    {
+      id: 'recovery-choice-phone',
+      value: CHOICES.phone,
+      image: <BackupRecoveryPhoneSmsImage />,
+      localizedChoiceTitle: ftlMsgResolver.getMsg(
+        'signin-recovery-method-phone',
+        'Recovery phone'
+      ),
+      // This doesn't need localization
+      localizedChoiceInfo: `••••••${lastFourPhoneNumber}`,
```

Comment:
This direction was decided: https://mozilla.slack.com/archives/C07T1GHL46S/p1735860376650479?thread_ts=1735835396.308079&cid=C07T1GHL46S